client/src/landing/AnonymousHome.js

```diff
@@ -0,0 +1,364 @@
+          This makes it possible to connect the dots, no matter where they lead.
+        </p>
+      </Col>
+    </Row>
+  </div>;
+}
+
+function TutorialLink(props) {
+  const url = props.url;
+  if ((url == null) || (url.length < 1))
+    return null;
+
+  if (url.startsWith("http")) {
+    return <ExternalLink
+      title="Follow the tutorial"
+      className="btn btn-outline-info" role="button" id="link-learn"
+      showLinkIcon={true}
+      url={url} />;
+  }
+  return <Link className="btn btn-outline-info" role="link" id="link-tutorial" to={url}>
+    Follow the tutorial
+  </Link>;
+}
+
+function Section4(props) {
+  const backgroundUrl = VisualDetail;
+  return <div id="rk-anon-home-section4"
+    style={{
+      backgroundImage: `url(${backgroundUrl})`
+    }}>
+    <div className="rk-w-s">
+      <div className="rk-pt-l">
+        <h3 className="text-info">Connecting dots</h3>
+      </div>
+      <div className="rk-pt-s">
+        <h3 className="text-white">The knowledge graph powers Renku and helps you make sense of what has been done.</h3>
+      </div>
+      <div className="rk-pt-s">
+        <h3 className="text-secondary">Ready to try it out?</h3>
+        <h3 className="text-info">Get started with Renku</h3>
+      </div>
+    </div>
+    <div className="d-flex flex-wrap">
+      <div className="pt-2" style={{ minWidth: "160px" }}>
+        <span>
+          <Link className="btn btn-outline-secondary me-1" role="button" id="link-sign_up" to="/login">
+        &nbsp;Sign Up
+          </Link>
+          (It&apos;s free)
+        </span>
+      </div>
+      <div className="pt-2" style={{ minWidth: "185px" }}>
+        <TutorialLink url={props.tutorialLink} />
+      </div>
+      <div className="pt-2" style={{ minWidth: "180px" }}>
+        <ExternalLink
+          title="Learn more"
+          className="btn btn-outline-info" role="button" id="link-learn"
+          showLinkIcon={true}
+          url="https://renku.readthedocs.io/en/latest/" />
+      </div>
+    </div>
+  </div>;
+}
+
+function Section5(props) {
+  return (props.projects == null) || (props.projects.length < 1) ?
+    <div id="rk-anon-home-section5-empty">
+      <div className="rk-pt-l">
+        <h3 className="text-info">&nbsp;</h3>
+      </div>
+    </div> :
+    <div id="rk-anon-home-section5">
+      <div className="rk-pt-l">
+        <h3 className="text-info">Look at some example projects</h3>
+      </div>
+    </div>;
+}
+
+function Section6(props) {
+  const backgroundUrl = VisualFooter;
+  return <div id="rk-anon-home-section6"
+    style={{
+      backgroundImage: `url(${backgroundUrl})`
+    }}>
+    <div>
+      <div><img src={logo} alt="Renku" height="68" className="d-block my-1" /></div>
+      <Row className="rk-pt-s" >
+        <Col xs={12} xl={4} >
+          <p>We have offices in both Lausanne on the EPFL campus and in Zürich at ETH Zürich.</p>
+        </Col>
+        <Col xs={12} lg={5} xl={4} className="rk-pl-s bg-primary">
```

Comment:
```suggestion
        <Col xs={12} lg={5} xl={4} className="rk-pt-s bg-primary">
```

The extra space on left (and no space on top) makes the output on medium/small screen very close to the above content and misaligned on the left (as in screen 1).
The suggested correction (screen 2) works, but the `xl={4}` at line 293 should also get removed. Otherwise, the content is better on smaller sizes but it's misaligned on bigger.

![Screenshot_20210416_140350](https://user-images.githubusercontent.com/43481553/115023961-bd7d9c00-9ebf-11eb-8ac0-83dd46343904.png)

![Screenshot_20210416_140608](https://user-images.githubusercontent.com/43481553/115024013-cd957b80-9ebf-11eb-903c-40cba57d1c3a.png)

Reply:
So the designer had this for the layout on desktop:

![image](https://user-images.githubusercontent.com/1196411/115037207-8367c680-9ece-11eb-9eb1-ad82c12408ce.png)

So I want to maintain that for the larger sizes, but I now have this:

![image](https://user-images.githubusercontent.com/1196411/115037328-a4301c00-9ece-11eb-9dbc-81f0ac5502e1.png)

and this

![image](https://user-images.githubusercontent.com/1196411/115037384-b1e5a180-9ece-11eb-98b8-93b40aa16b4f.png)

for smaller sizes.